internal/godocfx/testdata/golden/index.yml

```diff
@@ -288,6 +288,7 @@ items:
   - cloud.google.com/go/storage.Writer.Write
   - cloud.google.com/go/storage.SignedURL
   alt_link: https://pkg.go.dev/cloud.google.com/go/storage
+  friendlyApiName: Storage API
```

Comment:
This looks good! This should also be generated for child pages of the library as well.

Reply:
Which pages are you referring to? For Go, there is only one page per package.

Reply:
Actually, on closer look of Go API reference pages, it looks like there's one page per library 🤯. I don't think this applies to Go then.